kong-ngx-build

```diff
@@ -164,7 +166,6 @@ main() {
         echo "$OPENSSL_SHA openssl-$OPENSSL_VER.tar.gz" | sha256sum -c -
       fi
       tar -xzvf openssl-$OPENSSL_VER.tar.gz
-      ln -s openssl-$OPENSSL_VER openssl
```

Comment:
these symlinks were at one point used by downstream scripts which were unaware of the version of openssl / pcre we're using. I think that's no longer the case. For my own knowledge what was the issue with having them?

Reply:
1. If we ask for a different version of one of those dependencies in a subsequent run, and both versions are already downloaded, the symlink will not be pointing to the right version.
2. The PCRE symlink was invalid (pointing to a non-existing directory)
3. If we don't need them, let's get rid of them :)